aggregator/options.go

```diff
@@ -215,38 +131,14 @@ func NewOptions() Options {
 	return o
 }
 
-func (o *options) SetDefaultCounterAggregationTypes(aggTypes policy.AggregationTypes) Options {
+func (o *options) SetAggregationTypesOptions(value policy.AggregationTypesOptions) Options {
```

Comment:
nit: perhaps the aggregation types options should be grouped with other options (e.g., clock options etc)?

Reply:
sounds good